go.mod

```diff
@@ -15,6 +15,6 @@ require (
 	gorgonia.org/tensor v0.9.6
 	gorm.io/driver/mysql v1.0.3
 	gorm.io/driver/postgres v1.0.5
-	gorm.io/driver/sqlite v1.1.3
+	gorm.io/driver/sqlite v1.1.4-0.20201029040614-e1caf3738eb9
```

Comment:
Replace this too v1.1.4 after released.

https://github.com/go-gorm/sqlite/pull/20#issuecomment-718349434